docs/QA_Checklist.md

```diff
@@ -0,0 +1,59 @@
+# QA Checklist
+
+## Sandbox account credentials
+
+If you are using the PayPal or Stripe sandbox accounts in [sample.env](../sample.env) then you can use the following credentials/credit cards for QA
+
+### Paypal accounts
+
+send-donation@test.com / testtest
```

Comment:
Is it clear that this is name/password?

Why not just do 
name: send-donation@test.com
password: testtest